package com.testcase;

import org.testng.annotations.Test;

public class Demo_RecordAfterPutin1 {

    @Test
    public void putIntA() throws Exception{
        System.out.println("装入A ");
        System.out.println("1号小哥记录");
    }

    @Test
    public void putIntB() throws Exception{
        System.out.println("装入B ");
        System.out.println("2号小哥记录");
    }


    @Test
    public void putIntC() throws Exception{
        System.out.println("装入C ");
        System.out.println("3号小哥记录");
    }

    @Test
    public void putIntD() throws Exception{
        System.out.println("装入D ");
        System.out.println("4号小哥记录");
    }

    @Test
    public void putIntE() throws Exception{
        System.out.println("装入E ");
        System.out.println("5号小哥记录");
    }

}
